textattack/transformations/word_swap_change_number.py

```diff
@@ -0,0 +1,161 @@
+from num2words import num2words
+import numpy as np
+from word2number import w2n
+
+from textattack.transformations import Transformation
+
+
+def idx_to_words(ls, words):
+    """Given a list generated from cluster_idx, return a list that contains
+    sub-list (the first element being the idx, and the second element being the
+    words corresponding to the idx)"""
+
+    output = []
+    for cluster in ls:
+        word = words[cluster[0]]
+        for idx in cluster[1:]:
+            word = " ".join([word, words[idx]])
+        output.append([cluster, word])
+    return output
+
+
+def cluster_idx(idx_ls):
```

Comment:
(see above comment about replacing `cluster_idx` with a call to `mit.consecutive_groups`)